### PR TITLE
Bevy 0.14

### PR DESCRIPTION
**Pull request by rparrett**
The only complication was https://github.com/dimforge/bevy_rapier/issues/571, but an adequate workaround was found.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -375,16 +387,16 @@ fn decorate_track(
...
+                    extract_mesh_vertices_indices(meshes.get(mesh_handle).unwrap()).unwrap();
+                let collider: Collider = SharedShape::trimesh_with_flags(vtx, idx, flags).into();
+
                 commands
                     .entity(mesh_entity)
-                    .insert(ColliderDebugColor(Color::GREEN))
-                    .insert(
-                        Collider::from_bevy_mesh(
-                            meshes.get(mesh_handle).unwrap(),
-                            &ComputedColliderShape::TriMesh,
```
Comment (ThierryBerger) [inline src/main.rs:384]: Would this have been possible? @rparrett 

```rs
&ComputedColliderShape::TriMesh(TriMeshFlags::FIX_INTERNAL_EDGES),
```

Comment (rparrett) [reply src/main.rs:384]: [`ComputedColliderShape::TriMesh`](https://docs.rs/bevy_rapier3d/0.27.0/bevy_rapier3d/geometry/enum.ComputedColliderShape.html) doesn't seem to contain a field.

Comment (rparrett) [reply src/main.rs:384]: Ah, I see, this was added very recently and is not in a release yet.

Comment (ThierryBerger) [reply src/main.rs:384]: oh ok my bad thanks for checking it out

Comment (rparrett) [reply src/main.rs:384]: No worries, appreciate you diving in to help.

**Pull request merged**